internal/pkg/build/sources/packer_squashfs.go

```diff
@@ -37,24 +39,28 @@ func (p *SquashfsPacker) Pack() (*types.Bundle, error) {
 
 // unpackSquashfs removes the image header with dd and then unpackes image into bundle directories with unsquashfs
 func (p *SquashfsPacker) unpackSquashfs(b *types.Bundle, info *loop.Info64, rootfs string) (err error) {
+	var stderr bytes.Buffer
+
 	trimfile, err := ioutil.TempFile(p.b.Path, "trim.squashfs")
+	if err != nil {
+		return fmt.Errorf("While making tmp file: %v", err)
+	}
 
 	//trim header
 	sylog.Debugf("Creating copy of %s without header at %s\n", rootfs, trimfile.Name())
 	cmd := exec.Command("dd", "bs="+strconv.Itoa(int(info.Offset)), "skip=1", "if="+rootfs, "of="+trimfile.Name())
-	err = cmd.Run()
-	if err != nil {
-		sylog.Errorf("Trimming header Failed: %s", err)
-		return err
+	cmd.Stderr = &stderr
+	if err := cmd.Run(); err != nil {
+		return fmt.Errorf("Trimming header Failed: %v: %v", err, stderr.String())
 	}
 
 	//copy filesystem into bundle rootfs
```

Comment:
I may be weird, but I would put a space there for consistency: (same for line `49`, or `42` depending on how you look at it :smile:)

```suggestion
 	// copy filesystem into bundle rootfs
```

Reply:
Will do :+1: